src/compiler/abstract_compiler.nit

```diff
@@ -69,6 +69,8 @@ redef class ToolContext
 	var opt_no_gcc_directive = new OptionArray("Disable a advanced gcc directives for optimization", "--no-gcc-directive")
 	# --release
 	var opt_release = new OptionBool("Compile in release mode and finalize application", "--release")
+	# -g
+	var opt_debug = new OptionBool("Compile in debug mode (removes C-side optimization)", "--debug", "-g")
```

Comment:
This is only a tradition, but we used to put the short option before the long one: `"-g"`, `"--debug"`